app/configurator/configurator-state.tsx

```diff
@@ -1604,12 +1612,14 @@ const ConfiguratorStateProviderInternal = ({
               console.error(e);
               dispatch({ type: "PUBLISH_FAILED" });
             }
-          })();
-          return;
+            return;
+          }
+        }
+      } catch (e) {
+        console.error(e);
       }
-    } catch (e) {
-      console.error(e);
-    }
+    };
+    run();
   }, [state, dispatch, chartId, push, asPath, locale, query.from, replace]);
```

Comment:
Nit: I think we do not need to update the chart in the db when locale changes.

Reply:
True, fixed in 350744ca46f1ae350e47c8fa47049f34ae3befe3, the dependencies were not up to date.